internal/build/image_builder.go

```diff
@@ -45,6 +47,17 @@ func DefaultBuilder(b *dockerImageBuilder) ImageBuilder {
 	return b
 }
 
+func DefaultConsole() console.Console {
```

Comment:
you should take the io.Writer as a parameter here and let wire take care of it

Reply:
so, `ConsoleFromFile` wants an `*os.File` so I don't think that works